akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -469,6 +468,11 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
    */
   private[cluster] object ClusterLogger extends ClusterLogger(log)
 
+  /**
+   * INTERNAL API
+   */
+  private[cluster] object NoOpClusterLogger extends ClusterLogger(NoMarkerLogging)
```

Comment:
is this used?

Reply:
ah that snuck in. I was going down a different route at one point